libs/wingc/src/jsify.rs

```diff
@@ -604,10 +606,6 @@ impl JSifier {
 			None => ("", "=> "),
 		};
 
-		let async_prefix = match phase {
```

Comment:
Is this intentional

Reply:
It's unused

Reply:
It's dead code. The compiler wars about it. Initially I commented it out but @MarkMcCulloh suggested just removing it since it's no biggy to rewrite when we'll need it.